package.json

```diff
@@ -17,7 +18,9 @@
     "typescript": "^5.0.4"
   },
   "dependencies": {
-    "@ethereum-attestation-service/eas-contracts": "^0.26.1",
-    "@openzeppelin/contracts": "^4.8.2"
+    "@nomicfoundation/hardhat-ethers": "^3.0.4",
```

Comment:
Isn't this the newer version of `"@nomiclabs/hardhat-ethers": "^2.0.2",` ?
Do we need both?

Reply:
Good call! We were able to remove the duplicate